### ISSUE TITLE
Hierarchy handling mode 'Include Children' puts everything in a single layer with broken edge routing

### ISSUE THREAD

**Issue by NiklasRentzCAU** [eclipse/elk#700]
The hierarchy handling mode 'include children' creates broken layouts with all nodes in a single layer and broken edge routing since ELK 0.7.0.
An example is compiling the following SCChart for example with Netlist-based C and selecting the Basic Blocks step of the SCG:

```
scchart Test {
	input signal I
	output signal O

	region {
		initial state A
		if I do O go to B

		final state B
	}
}
```
This is the resulting layout using ELK 0.7.0:
![Screenshot from 2020-09-14 16-22-16](https://user-images.githubusercontent.com/47597619/93097980-91d6d100-f6a6-11ea-9159-0d06d6d2d869.png)
And this is what is expected (generated while turning the option off, therefore using the 'separate children' option):
![Screenshot from 2020-09-14 16-31-04](https://user-images.githubusercontent.com/47597619/93098957-c008e080-f6a7-11ea-8921-57cff8868fb6.png)

**Comment by uruuru** [comment]
Can you please add the underlying elk graph or a minimal example to reproduce?

**Comment by NiklasRentzCAU** [comment]
Not minimal, but the elkg logged from the layout graph view from the example above:
[1600097370229gw-Diagram_layout_engine.zip](https://github.com/eclipse/elk/files/5219452/1600097370229gw-Diagram_layout_engine.zip)

**Comment by uruuru** [comment]
Thanks. Setting `hierarchyHandling: INCLUDE_CHILDREN` for the _real root node_ (not the container node) seems to help. 

Still, this seems to be a bug as it should be just fine as it is configured.

**Comment by uruuru** [comment]
I believe the issue was introduced in 66f943f995f521795e78b23e01bf9de5ce0a2396 (#665). The condition used to decide whether to transform edges results in no edges being imported at all for the given graph (where hierarchy handling is not activated for the root but for the first child of the root). 

The workaround I mentioned above, should hence work fine.

**Comment by soerendomroes** [comment]
If this is fixed in the next week or so, we could consider to release a patch together with the fix for #696
